### ISSUE TITLE
[Fleet]: Last activity 'in x seconds ago' is shown for Mac Fleet Server agent.

### ISSUE THREAD

**Issue by simranvaseer-qasource** [elastic/fleet-server#1704]
**Kibana version**: 8.4 BC1 Kibana cloud environment

**Host OS and Browser version**: All, All

**Build details:**

VERSION: 8.4 BC1
BUILD: 54999
COMMIT: 58f7eaf0f8dc3c43cbfcd393e587f155e97b3d0d
ARTIFACT LINK: https://staging.elastic.co/8.4.0-91109cf0/summary-8.4.0.html

**Preconditions:**
- 8.4 BC1 Kibana cloud environment should be available.

**Steps to reproduce:**

1. Login to Kibana environment.
2. Now, Navigate to Agents tab.
3. Click on 'Add Fleet Server' button.
4. Add Fleet Server host say' http://10.10.10.10:8220/
5. Click on 'Generate Fleet server policy'
6. Copy and run the command on cli.
7. Observe the last activity of fleet server agent 'in x seconds ago'.

**Expected Result:**
Last activity 'x seconds ago' should be shown for Mac Fleet Server agent.

**Screenshots:**
![1](https://user-images.githubusercontent.com/102019374/182113618-39d3666f-eeba-481a-9e47-64cf133642ed.PNG)

**Comment by simranvaseer-qasource** [comment]
@dikshachauhan-qasource Please review

**Comment by dikshachauhan-qasource** [comment]
Secondary review done for this ticket.

**Comment by jlind23** [comment]
@kpollich I just came across this issue but it seems to be a fleet ui one and not a fleet server. May I transfer it?

**Comment by kpollich** [comment]
Yes this is probably a calculation issue in Fleet UI - let's transfer it
